client-common/Cargo.toml

```diff
@@ -40,16 +40,12 @@ tokio = { version = "0.2", features = ["rt-threaded", "sync", "time", "tcp"], op
 tokio-tungstenite = { version = "0.10", features = ["tls"], optional = true }
 uuid = { version = "0.8.1", features = ["v4"] }
 webpki = "0.21"
-yasna = { version = "0.3.1", features = ["bit-vec", "num-bigint", "chrono"], optional = true }
 zeroize = "1.1"
 
 [dev-dependencies]
 quickcheck = "0.9"
 
 [features]
-default = ["sled", "websocket-rpc"]
+default = ["sled", "websocket-rpc", "ra-client"]
```

Comment:
We can remove `optional = true` from `ra-client` dependency and remote it from features also.

```suggestion
default = ["sled", "websocket-rpc"]
```

Reply:
ok, I guess it won't affect the mock mode?